contracts/colony/Colony.sol

```diff
@@ -48,6 +48,27 @@ contract Colony is ColonyStorage, PatriciaTreeProofs, MultiChain {
   function makeArbitraryTransaction(address _to, bytes memory _action)
   public stoppable auth
   returns (bool)
+  {
+    return makeArbitraryTransactionFunctionality(_to, _action);
+  }
+
+  function makeArbitraryTransactions(address[] memory _targets, bytes[] memory _actions)
+  public stoppable auth
+  returns (bool)
+  {
+    require(_targets.length == _actions.length, "colony-targets-and-actions-length-mismatch");
+    for (uint256 i; i < _targets.length; i+= 1){
+      require(
```

Comment:
Should these be wrapped in a `require`? In other uses, we allow transactions to fail and return a `false` bool. At minimum, let's consider adding a `strict` flag which throws an error if any tx fails, otherwise let's return a bool, making it a user decision.

Reply:
I can get on board with a flag.